File: Zadania/2025/Zadanie-6/Python/Druzyna.py
from Wojownik import Wojownik

class Druzyna:
    def __init__(self , nazwaDruzyny ):
        self.nazwaDruzyny = nazwaDruzyny
        self.czlonkowie = [ Wojownik ]

    def dodajWojownika(self , wojownik):
        self.czlonkowie.append(wojownik)

File: Zadania/2025/Zadanie-6/Java/Druzyna.java
package Java;

import java.util.ArrayList;
import java.util.List;

public class Druzyna {
    String nazwaDruzyny;
    List<Wojownik> czlonkowie;

    Druzyna(String nazwaDruzyny, List<Wojownik> czlonkowie) {
        this.nazwaDruzyny = nazwaDruzyny;
        this.czlonkowie = new ArrayList<Wojownik>();
    }
    public void dodajWojownika(Wojownik wojownik){
        this.czlonkowie.add(wojownik);
    }
    public List<Wojownik> getCzlonkowie() { return czlonkowie; }
}
